src/webgpu/util/math.ts

```diff
@@ -919,11 +919,15 @@ export function filteredF64Range(
 /** Short list of i32 values of interest to test against */
 const kInterestingI32Values: number[] = [
   kValue.i32.negative.max,
-  kValue.i32.negative.max / 2,
+  Math.trunc(kValue.i32.negative.max / 2),
+  -256,
+  -10,
   -1,
   0,
   1,
-  kValue.i32.positive.max / 2,
+  10,
+  256,
+  Math.trunc(kValue.i32.positive.max / 2),
```

Comment:
Why do we need to truncate these?

Reply:
We chatted about this OOB, I am including the summary of it here, for future reference.

These values are odd integers, so when divided return a fractional result, which is stored in the `number` since it is a f64 under the hood. This leads to other calculations ending up slightly off and the test failing.
The changes that I made to the test generation code should catch this and quantize things correctly, but it feels weird to have a known non-integer in a list of nominally integer value.s